src/app/Data/Migrations/20190708000727_Inital.cs

```diff
@@ -0,0 +1,220 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace app.Data.Migrations
```

Comment:
idiom?
the usual idiom for C# is initial casing
is there sufficient reason to not do this?

Reply:
Not discussed this yet as a group, how we're going to namespace. We'll likely stick to style standards.